docs/src/main/sphinx/connector/cassandra.rst

```diff
@@ -5,10 +5,18 @@ Cassandra connector
 The Cassandra connector allows querying data stored in
 `Apache Cassandra <https://cassandra.apache.org/>`_.
 
-Compatibility
--------------
-
-Connector is tested against Cassandra version 2.2 and 3.11, but any intermediate or newer versions are expected to work.
+Requirements
+------------
+
+Requirements for using the connector in a catalog to connect to a Cassandra
+data source are:
+
+* Cassandra version 2.2 or higher or PostgreSQL 9.6 or higher
+* Network access from the Trino coordinator and
+  workers to Cassandra:
+  * For cluster communication, default on port 7000
+  * For native protocol client access, default on port 9042
+  * For JMX, default to port 7199
```

Comment:
I think we should document only 9042 port.

Reply:
yeah .. I wasnt sure. Thanks. Fixed now.